lumigator/python/mzai/backend/backend/api/routes/completions.py

```diff
@@ -5,17 +5,20 @@
 
 router = APIRouter()
 
+VENDOR_MISTRAL = "mistral"
+VENDOR_OPENAI = "openai"
+
 
 @router.get("/")
 def list_vendors():
-    return ["openai", "mistral"]
+    return [VENDOR_MISTRAL, VENDOR_OPENAI]
 
 
-@router.post("/mistral")
+@router.post(f"/{VENDOR_MISTRAL}")
```

Comment:
Later on, I'd use a more dynamic dict[vendor_name:str, vendor_service: CompletionService] that is stored persistently.